.gitignore

```diff
@@ -1 +1,3 @@
 /node_modules/
+/.idea/
```

Comment:
What is this for? Intellij?

Reply:
It's from Jetbrain's Webstorm.

Reply:
Can you add this to your own global gitignore maybe?
We probably don't want add this for every IDE.